organizational/position_roles.rst

```diff
@@ -0,0 +1,40 @@
+Position Roles 
+========================
+
+Below are the positions that have allowed Hacker's Society function in the past. The
```

Comment:
Grammar seems off.
"allowed Hacker Society to function" or "allowed Hacker's Society functioning" or something of the sort